django/contrib/gis/db/models/__init__.py

```diff
@@ -11,4 +11,4 @@
 from django.contrib.gis.db.models.fields import (  # NOQA
     GeometryField, PointField, LineStringField, PolygonField,
     MultiPointField, MultiLineStringField, MultiPolygonField,
-    GeometryCollectionField)
+    GeometryCollectionField, RasterField)
```

Comment:
That was the wrong link, sorry!

https://github.com/geodesign/django/blob/raster_field/django/contrib/gis/db/models/fields.py#L406

With that, the dummy field can be used similarly to the GeometryField when imported in the tests:

https://github.com/django/django/pull/4339/files#diff-da8fd5c3308a451e84113f8a4a810457